src/db/cass/create_db.py

```diff
@@ -58,6 +58,9 @@
 # Table for storing Notifications
 cluster_services.create_table_schema('CREATE TABLE IF NOT EXISTS notif (date timestamp, type text, email text, name text, frame_id text, stream_id text, PRIMARY KEY(date))')
 
+# Table for Blacklist
+cluster_services.create_table_schema('CREATE TABLE IF NOT EXISTS blacklist (name text), PRIMARY KEY(name))')
```

Comment:
![Screenshot from 2020-05-06 22-37-12](https://user-images.githubusercontent.com/21010314/81248411-4884bf80-8fea-11ea-8393-8cb769ec0dfc.png)
I got this error from this

Reply:
Had an extra parenthesis there. Shouldn't have this issue now.